Sources/StreamChat/Config/TokenProvider_Tests.swift

```diff
@@ -17,15 +17,16 @@ final class TokenProvider_Tests: StressTestCase {
 
     func test_developmentProvider_propagatesToken() throws {
         // Create a user identifier.
-        let userId: UserId = "luke_skywalker"
+        let userId: UserId = .unique
 
         // Get a token from `development` token provider.
         let token = try await {
             TokenProvider.development(userId: userId).getToken(.mock, $0)
         }.get()
 
         // Assert token is correct.
-        let expectedToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoibHVrZV9za3l3YWxrZXIifQ==.devtoken"
+        let expectedPayload = #"{"user_id":"\#(userId)"}"# .data(using: .utf8)!.base64EncodedString()
+        let expectedToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.\(expectedPayload).devtoken"
```

Comment:
(feel free to ignore) nit: The `user_id` payload is already covered by `XCTAssertEqual(token.userId, userId)`. Here you could do just:
```swift
let tokenParts = token.split(separator: ".")
XCTAssertEqual(tokenParts[0], "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9")
XCTAssertEqual(tokenParts[2], "devtoken")
```